web/ui/module/lezer-promql/src/promql.grammar

```diff
@@ -275,6 +279,11 @@ NumberLiteral  {
   }
   Identifier { (std.asciiLetter | "_" | ":") (std.asciiLetter | std.digit | "_" | ":" )*}
   LabelName { (std.asciiLetter | "_") (std.asciiLetter | std.digit | "_")* }
+  QuotedLabelName {
+    '"' (![\\\n"] | "\\" _)* '"' |
+    "'" (![\\\n'] | "\\" _)* "'" |
+    "`" ![`]* "`"
+  }
```

Comment:
Why not writing the rule `QuotedLabelName` like that:

```
QuotedLabelName {
 '"'  LabelName  '"' | 
 "'"  LabelName "'" |
 "`" LabelName "`"
}
```

Looks more accurate since a QuotedLabelName is labelName between quote.

Reply:
Yeah, we can reduce `QuotedLabelName` to avoid repeatation. But writing it in the form of : 
```
QuotedLabelName {
 '"'  LabelName  '"' | 
 "'"  LabelName "'" |
 "`" LabelName "`"
}
```
is incorrect as `QuotedLabelName` can have all the `utf-8` chars but `LabelName` does not support that. 
I think a better way will be like this:
```
QuotedLabelName {
  StringLiteral
}
```
wdyt?

Reply:
ah I forgot that `QuotedLabelName` support `utf-8`, sorry about that. In that case yeah your last suggestion makes totally sense !

Once you changed, we should be good to merge this PR

Reply:
Yeah, I've reduced `QuotedLabelName` with `StringLiteral` I think it is ready for the final review.